tooling/project-template/src/main/resources/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```

Comment:
If we commit gradlew, should we provide gradlew.bat as well?

Reply:
I would agree, were it not for the fact that in my latest correction I actually deleted it, because it actually doesn't work any more ('gradle wrapper' update run is needed, which defeats the purpose of having it in the first place a bit).
Either way I did not amend that commit correctly, so it didn't actually get removed; I noticed thanks to your comment.
I've corrected the mistaken commit. Thanks!